drivers/include/mrf24j40.h

```diff
@@ -133,6 +132,7 @@ typedef struct {
     netdev_ieee802154_t netdev;             /**< netdev parent struct */
     /*  device specific fields  */
     mrf24j40_params_t params;               /**< parameters for initialization */
+    uint16_t flags;                         /**< Internal device flags */
```

Comment:
and make this an `uint8_t` which would be enough for the flags above.

Reply:
also changed